tests/server/TestLocalesConfig.js

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs';
+import { assert } from 'chai';
+import path from 'path';
+import config from 'config';
+import { langToLocale, localeToLang } from 'core/i18n/utils';
+import glob from 'glob';
+
+const langs = config.get('langs');
+const basePath = config.get('basePath');
+
+describe('Locale Config', () => {
+  for (const lang of langs) {
```

Comment:
If you use `langs.forEach(() => {` you should be able to avoid the warning. I don't think there's a bug here but it's probably safer. Same for below.